bindings/src/Capgemini.PowerApps.SpecFlowBindings/Configuration/TestConfiguration.cs

```diff
@@ -18,6 +19,9 @@ public class TestConfiguration
 
         private const string GetUserException = "Unable to retrieve user configuration. Please ensure a user with the given alias exists in the config.";
 
+        private readonly object userEnumeratorsLock = new object();
+        private readonly ThreadLocal<UserConfiguration> currentUser = new ThreadLocal<UserConfiguration>();
```

Comment:
For consistency with what we've got in the `PowerAppsStepDefiner` class I would've resolved the SonarCloud comment by adding the `static` keyword that was missing rather than using a `ThreadLocal<>` instance member.